### DISCUSSION TITLE
Radarr Update killed Radarr

### DISCUSSION THREAD

**Discussion by sambarlick** [tteck/Proxmox#2040] [Help Needed] [answered]
Hi,

I ran the command update to update Radarr and now Radarr won't start.

This is what is displayed in the browser when attempting to access the web GUI.

{
  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.4",
  "title": "Not Found",
  "status": 404,
  "traceId": "00-76a133a6b1f7e6caceafa3c1e094c85b-cc0291e23d63665c-00"
}

Sam

**Comment by tteck** [comment, marked as answer]
Strange, because the instructions give no update method
![Screenshot 2023-11-12 10 06 39 AM](https://github.com/tteck/Proxmox/assets/21342670/b967ebc2-d0e8-42c2-8f90-11395212a735)
and if you try, it only does a 
```
function update_script() {
header_info
if [[ ! -d /var/lib/radarr/ ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
msg_info "Updating $APP LXC"
apt-get update &>/dev/null
apt-get -y upgrade &>/dev/null
msg_ok "Updated $APP LXC"
exit
}
```
So, your issue is not with the update.

**Reply by sambarlick** [reply]
Ok thanks.
